app/src/main/java/app/model/servicedefinition/ServiceDefinition.java

```diff
@@ -0,0 +1,75 @@
+// Copyright 2023 Libre311 Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app.model.servicedefinition;
+
+
+import app.model.service.Service;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "service_definitions")
+public class ServiceDefinition {
```

Comment:
One thing I'm trying to wrap my head around is why we need a ServiceDefintion table?  Cant ServiceDefinitionAttribute have the service_id as a foreign key? 
ie:
```
class Service {
    @OneToMany
    private List<ServiceDefinitionAttribute> serviceDefinitionAttributes;
}
```

```
ServiceDefinitionAttribute {
    @ManyToOne(optional = false)
    private Service service;
}
```
Mentioning out of curiosity, not advocating that it _should_ be one way or the other

Reply:
The approach is a relic from the previous implementation using DTO but I like the idea of dropping the service_definition table.